src/main/java/org/jabref/JabRefException.java

```diff
@@ -29,11 +32,13 @@ public JabRefException(String message, String localizedMessage, Throwable cause)
     public JabRefException(Throwable cause) {
         super(cause);
     }
-
+     
+    // To display localized message on error
     @Override
     public String getLocalizedMessage() {
+        // If there is no localized message
         if (localizedMessage == null) {
```

Comment:
No added value as if clause is self-explanatory.

Reply:
Okay sir.